core/cloudflow-operator/src/main/scala/cloudflow/operator/Main.scala

```diff
@@ -96,13 +95,6 @@ object Main extends {
 
   private def exitWithFailure() = System.exit(-1)
 
-  private def createMaterializer()(implicit system: ActorSystem) = {
-    val decider: Supervision.Decider = {
-      case _ ⇒ Supervision.Stop
-    }
-    ActorMaterializer(ActorMaterializerSettings(system).withSupervisionStrategy(decider))
-  }
-
```

Comment:
@RayRoestenburg does this change the behavior when a stream crashes? Or is `Stop` the default now?

Reply:
ah, forget it, I just noticed it got moved to custom attributes 😉